src/sickle.ts

```diff
@@ -66,9 +66,31 @@ class Annotator {
         this.writeNode(classNode.getLastToken());
         break;
       }
+      case ts.SyntaxKind.PublicKeyword:
```

Comment:
I considered this, but my best guess is that we don't, because there's no reason to @export a private variable.  What do you think?